user/probe_gnutls.go

```diff
@@ -99,14 +99,19 @@ func (e *MGnutlsProbe) constantEditor() []manager.ConstantEditor {
 
 func (this *MGnutlsProbe) setupManagers() error {
 	var binaryPath string
+	_, e := os.Stat(X86_BINARY_PREFIX)
+	prefix := X86_BINARY_PREFIX
+	if e != nil {
+		prefix = OTHERS_BINARY_PREFIX
+	}
 	switch this.conf.(*GnutlsConfig).elfType {
 	case ELF_TYPE_BIN:
 		binaryPath = this.conf.(*GnutlsConfig).Curlpath
 	case ELF_TYPE_SO:
 		binaryPath = this.conf.(*GnutlsConfig).Gnutls
 	default:
 		//如果没找到
-		binaryPath = "/lib/x86_64-linux-gnu/libgnutls.so.30"
+		binaryPath = prefix + "/libgnutls.so.30"
```

Comment:
keep origin value due `binaryPath` is certain from `config_*` struct.

Reply:
> keep origin value due `binaryPath` is certain from `config_*` struct.

Sorry I missed to understand this comment, why the logic should change here?